src/main/java/org/apache/sling/graphql/core/engine/DefaultQueryExecutor.java

```diff
@@ -232,119 +211,6 @@ public ValidationResult validate(@NotNull String query, @NotNull Map<String, Obj
         }
     }
 
-    private RuntimeWiring buildWiring(TypeDefinitionRegistry typeRegistry, Iterable<GraphQLScalarType> scalars, Resource r) {
```

Comment:
The diffs for this file are quite large, I don't think they reflect the actual changes - can you easily revisit to create more precise diffs?